img/signals/signal.php

```diff
@@ -250,13 +252,14 @@
 	<!ENTITY ns_xlink "http://www.w3.org/1999/xlink">	
 	<!ENTITY background "fill:#000000;">	
 	<!ENTITY white "fill:#FFFFFF;">	
-	<!ENTITY gray "fill:#444444;">	
-	<!ENTITY red "fill:red;">	
-	<!ENTITY yellow "fill:yellow;">	
-	<!ENTITY green "fill:#38FFF5;">
-		
+	<!ENTITY gray "fill:#444444;">
+<?php
+	echo("<!ENTITY red "fill:" . $color_red . ";\">");	
+	echo("<!ENTITY yellow "fill:" . $color_yellow . ";\">");
```

Comment:
This should be `echo('<!ENTITY yellow "fill:' . $color_yellow . ';\">');`
You also do not need the brackets.